general/src/main/java/com/halfapum/general/coroutines/exception/ExceptionPropagator.kt

```diff
@@ -0,0 +1,39 @@
+package com.halfapum.general.coroutines.exception
+
+import android.app.Application
+import kotlinx.coroutines.channels.BufferOverflow
+import kotlinx.coroutines.flow.MutableSharedFlow
+import kotlinx.coroutines.flow.asSharedFlow
+
+object ExceptionPropagator {
+
+    private var _exceptionFlow: MutableSharedFlow<Throwable> = MutableSharedFlow(
+        10,
```

Comment:
Do not hardcode values, put it to constants, and please provide a documentation why exactly you chose these values as default